documentation/modules/rn-2.6.0.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+Migrate virtual machines (VMs) from VMware vSphere, {rhv-full}, or {osp} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+
+This release has the following technical changes:
```

Comment:
besides the aforementioned upgrades (if any), we can also include:
replacement of openshift templates with virtual machine preferences (https://issues.redhat.com/browse/MTV-206) - please work with @liranr23 on this, as we still fall to openshift templates when we can't find a relevant preference

I can't think of more items at the moment, but I'll add relevant ones when I'll review the items

Reply:
> besides upgrades from el8 to el9 (if any in his release), we can also include: MTV-206 - please work with @liranr23 on this, as we still fall to openshift templates when we can't find a relevant preference
> 
> I can't think of more items at the moment, but I'll add relevant ones when I'll review the items

If we can't find a relevant preference we fallback to our previous mechanism, using CNV templates.